File: Semestrul 4/MPP2/Client-Server_2Limbaje/Problema8SC_CSharp/Networking/dto/ChildEnrollDTO.cs
namespace Networking.dto;

using Problema8SC_CSharp.Model;

public class ChildEnrollDTO
{
    public Child Child { get; set; }
    public Enrollment Enrollment { get; set; }

    public ChildEnrollDTO(Child child, Enrollment enrollment)
    {
        Child = child;
        Enrollment = enrollment;
    }
}

File: Semestrul 4/MPP2/Problema8_SC_REST/Problema8SC_Rest/Problema8Networking/src/main/java/problema8/network/dto/ChildEnrollDTO.java
package problema8.network.dto;

import problema8.model.Child;
import problema8.model.Enrollment;
import problema8.model.Event;

public class ChildEnrollDTO {
    private Child child;
    private Enrollment enrollment;

    public ChildEnrollDTO(Child child, Enrollment enrollment) {
        this.child = child;
        this.enrollment = enrollment;
    }

    public Child getChild() {
        return child;
    }

    public void setChild(Child child) {
        this.child = child;
    }

    public Enrollment getEnrollment() {
        return enrollment;
    }

    public void setEnrollment(Enrollment enrollment) {
        this.enrollment = enrollment;
    }
}
